internal/process/update/update_runtime_step.go

```diff
@@ -0,0 +1,90 @@
+package update
+
+import (
+	"context"
+	imv1 "github.com/kyma-project/infrastructure-manager/api/v1"
+	"github.com/kyma-project/kyma-environment-broker/internal"
+	"github.com/kyma-project/kyma-environment-broker/internal/process"
+	"github.com/kyma-project/kyma-environment-broker/internal/storage"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"time"
+)
+
+type UpdateRuntimeStep struct {
+	operationManager *process.OperationManager
+	k8sClient        client.Client
+}
+
+func NewUpdateRuntimeStep(os storage.Operations, k8sClient client.Client) *UpdateRuntimeStep {
+	return &UpdateRuntimeStep{
+		operationManager: process.NewOperationManager(os),
+		k8sClient:        k8sClient,
+	}
+}
+
+func (s *UpdateRuntimeStep) Name() string {
+	return "Update_Runtime_Resource"
+}
+
+func (s *UpdateRuntimeStep) Run(operation internal.Operation, log logrus.FieldLogger) (internal.Operation, time.Duration, error) {
+	// Check if the runtime exists
```

Comment:
for reviewer: double check if all fields from UpdatingParametersDTO are copied to the runtimeSpec

Reply:
RuntimeAdministrators are not copied.